content/master/concepts/packages.md

```diff
@@ -1,513 +1,489 @@
-to `Inactive`. When `revisionActivationPolicy: Manual`, the user must manually
-edit a new revision and mark it as `Active`. This can be useful if you are using
-a `packagePullPolicy: Automatic` with a channel tag (e.g. `latest`) and you want
-Crossplane to create new revisions when a new version is available, but you
-don't want to automatically update to that newer revision.
-
-It is recommended for most users to use semver tags or image digests and
-manually update their packages, but use a `revisionActivationPolicy: Automatic`
-to avoid having to manually activate new versions. However, each user should
-consider their specific environment and choose a combination that makes sense
-for them.
-
-For security reasons, it's suggested using image digests instead or alongside
-tags (`vx.y.z@sha256:...`), to ensure that the package content wasn't tampered
-with.
-
-### spec.revisionHistoryLimit
-
-Valid values: any integer, disabled by explicitly setting to `0` (default `1`)
-
-When a revision transitions from `Inactive` to `Active`, its revision number
-gets set to one greater than the largest revision number of all revisions for
-its package. Therefore, as the number of revisions increases, the least recently
-`Active` revision will have the lowest revision number. Crossplane will garbage
-collect old `Inactive` revisions if they fall outside the
-`spec.revisionHistoryLimit`. For instance, if my revision history limit is `3`
-and I currently have three old `Inactive` revisions and one `Active` revision,
-when I upgrade the next time, the new revision will be given the highest
-revision number when it becomes `Active`, the previously `Active` revision will
-become `Inactive`, and the oldest `Inactive` revision will be garbage collected.
-
-> Note: In the case that `spec.revisionActivationPolicy: Manual` and you upgrade
-> enough times (but do not make `Active` the new revisions), it is possible that
-> activating a newer revision could cause the previously `Active` revision to
-> immediately be garbage collected if it is outside the
-> `spec.revisionHistoryLimit`.
-
-### spec.packagePullSecrets
-
-Valid values: slice of `Secret` names (secrets must exist in `namespace`
-Crossplane was installed in, typically `crossplane-system`)
-
-This field allows a user to provide credentials required to pull a package from
-a private repository on a registry. The credentials are passed along to a
-packaged controller if the package is a `Provider`, but are not passed along to
-any dependencies.
-
-### spec.skipDependencyResolution
-
-Valid values: `true` or `false` (default: `false`)
-
-If `skipDependencyResolution: true`, the package manager will install a package
-without considering its dependencies.
 
-### spec.ignoreCrossplaneConstraints
+### Verify a Configuration
 
-Valid values: `true` or `false` (default: `false`)
+Verify a Configuration with 
+{{<hover label="verify" line="1">}}kubectl get configuration{{</hover >}}.
```

Comment:
nit: The sentence feels a bit incomplete. Extending the sentence about expecting to see `Installed` and `Healthy` as `True` could help.